### PR TITLE
es: Update browser compat/spec sections (part 2)

### PR DESCRIPTION
**Pull request by queengooborg**
This PR updates some of the browser compatibility data and specification sections for the Spanish locale. Part of work for #11594.

Note: this may also include other minor updates, such as removal of old sections or other small bits of cleanup.

### CODE REVIEW

**Review by JuanVqz — APPROVED** [review]
Cumple con la convención. :+1: 

@queengooborg gracias!

**Comment by JuanVqz** [comment]
@Graywolf9 alguna opinion al respecto o podemos fusionar?

**Comment by Graywolf9** [comment]
Gracias por la revisión! @JuanVqz, faltaría revisar que todos muestran el contenido correctamente.

File: files/es/mozilla/add-ons/webextensions/manifest.json/index.md [hunk 1]
```diff
@@ -33,11 +33,9 @@ Puedes acceder a el manifest desde el código JavaScript de tu extensión utiliz
 browser.runtime.getManifest().version;
 ```
 
-## Compatibilidad de Navegadores
+## Compatibilidad con navegadores
 
-Para un resumen extendido de las llaves y sub-llaves ver la [tabla completa de compatibilidad de `manifest.json` en navegadores](/es/docs/Mozilla/Add-ons/WebExtensions/Browser_compatibility_for_manifest.json).
-
-{{Compat("webextensions.manifest")}}
+{{Compat}}
```
Comment (Graywolf9) [inline files/es/mozilla/add-ons/webextensions/manifest.json/index.md:38]: As `en-US`

```suggestion
Para un resumen extendido de las llaves y sub-llaves ver la [tabla completa de compatibilidad de `manifest.json` en navegadores](/es/docs/Mozilla/Add-ons/WebExtensions/Browser_compatibility_for_manifest.json).

{{Compat}}
```

File: files/es/web/api/canvasrenderingcontext2d/drawimage/index.md [hunk 2]
```diff
@@ -119,18 +119,9 @@ window.addEventListener("load", drawCanvas);
...
-- Empezando con (Firefox 5.0 / Thunderbird 5.0 / SeaMonkey 2.2) `drawImage()` maneja argumentos negativos de acuerdo con la especificación, volteando el rectangulo alrededor del eje apopiado.
-- Especificación de una imagen `null` o `undefined` al llamar o `drawImage()` correctamente lanzando una excepción `TYPE_MISMATCH_ERR` empezando con (Firefox 5.0 / Thunderbird 5.0 / SeaMonkey 2.2).
-- antes de Gecko 7.0 (Firefox 7.0 / Thunderbird 7.0 / SeaMonkey 2.4), Firefox lanzó una excepción si alguno de los valores de las coordenadas no eran finitos o cero. De acuerado a la especificación esto ya no ocurre.
-- Gecko 9.0 (Firefox 9.0 / Thunderbird 9.0 / SeaMonkey 2.6) ahora soporta correctamente CORS para dibujar imágenes a través de dominios sin [manchar el canvas](/en/CORS_Enabled_Image#What_is_a_.22tainted.22_canvas.3F).
-- Gecko 11.0 (Firefox 11.0 / Thunderbird 11.0 / SeaMonkey 2.8) ahora permite SVG-como-una-imagen para ser dibujada en el canvas sin [manchar el canvas](/en/CORS_Enabled_Image#What_is_a_.22tainted.22_canvas.3F).
+{{Compat}}
```
Comment (Graywolf9) [inline files/es/web/api/canvasrenderingcontext2d/drawimage/index.md:124]: As `en-US`

```suggestion
{{Compat}}

## Notas

-  `drawImage()` solo funciona correctamente en un {{domxref("HTMLVideoElement")}} cuando su {{domxref("HTMLMediaElement.readyState")}} es mayor que 1 (es decir, el evento de búsqueda se activa después de configurar la propiedad `currentTime).
- `drawImage()` siempre utilizará del elemento de origen _el tamaño intrínseco  en píxeles de CSS_ al dibujar, recortar o escalar.
- En algunas versiones anteriores del navegador, `drawImage()` ignorará todos los metadatos EXIF en las imágenes, incluida la Orientación. Este comportamiento es especialmente problemático en dispositivos iOS. Debe detectar la Orientación usted mismo y usar la función `rotate()` para corregirla.
```

File: files/es/web/api/domerror/index.md [hunk 3]
```diff
@@ -38,14 +38,14 @@ La interfaz **`DOMError`** describe un objeto de error que contiene un nombre de
...
+{{Specifications}}
 
-## Especificaciones
+## Compatibilidad con navegadores
 
-- [DOM 4 DOMError specification](https://www.w3.org/TR/dom/#interface-domerror)
+{{Compat}}
 
-## Véase también
+## Vér también
```
Comment (Graywolf9) [inline files/es/web/api/domerror/index.md:49]: According to `es` conventions

```suggestion
## Véase también
```

Comment (queengooborg) [reply files/es/web/api/domerror/index.md:49]: I've mostly seen "Vér" in place of "Véase" on pages, perhaps we should open an issue regarding that?

File: files/es/web/api/file/lastmodifieddate/index.md [hunk 4]
```diff
@@ -33,11 +33,11 @@ for (var i = 0; i < files.length; i++) {
 
 ## Especificaciones
 
-_Aunque estaba presente en las primeras versiones de la especificacion de File API, esta propiedad ha sido eliminada de esta y ahora es no-estandar. Usar {{domxref("File.lastModified")}} como reemplazo._
+{{Specifications}}
```
Comment (Graywolf9) [inline files/es/web/api/file/lastmodifieddate/index.md:36]: As `en-US`

```suggestion
_Aunque estaba presente en las primeras versiones de la especificacion de File API, esta propiedad ha sido eliminada de esta y ahora es no-estandar. Usar {{domxref("File.lastModified")}} como reemplazo._
```

File: files/es/web/api/domerror/index.md (hunk 3 above)
Comment (Graywolf9) [reply files/es/web/api/domerror/index.md]: I don't think an issue would be necessary, you see mostly "Vér" due to there is a lot of content outdated yet, that would be changed eventually

File: files/es/web/api/canvasrenderingcontext2d/drawimage/index.md [hunk 5]
```diff
@@ -119,18 +119,15 @@ window.addEventListener("load", drawCanvas);
...
-- Especificación de una imagen `null` o `undefined` al llamar o `drawImage()` correctamente lanzando una excepción `TYPE_MISMATCH_ERR` empezando con (Firefox 5.0 / Thunderbird 5.0 / SeaMonkey 2.2).
-- antes de Gecko 7.0 (Firefox 7.0 / Thunderbird 7.0 / SeaMonkey 2.4), Firefox lanzó una excepción si alguno de los valores de las coordenadas no eran finitos o cero. De acuerado a la especificación esto ya no ocurre.
-- Gecko 9.0 (Firefox 9.0 / Thunderbird 9.0 / SeaMonkey 2.6) ahora soporta correctamente CORS para dibujar imágenes a través de dominios sin [manchar el canvas](/en/CORS_Enabled_Image#What_is_a_.22tainted.22_canvas.3F).
-- Gecko 11.0 (Firefox 11.0 / Thunderbird 11.0 / SeaMonkey 2.8) ahora permite SVG-como-una-imagen para ser dibujada en el canvas sin [manchar el canvas](/en/CORS_Enabled_Image#What_is_a_.22tainted.22_canvas.3F).
+-  `drawImage()` solo funciona correctamente en un {{domxref("HTMLVideoElement")}} cuando su {{domxref("HTMLMediaElement.readyState")}} es mayor que 1 (es decir, el evento de búsqueda se activa después de configurar la propiedad `currentTime).
```
Comment (Graywolf9) [inline files/es/web/api/canvasrenderingcontext2d/drawimage/index.md:128]:
```suggestion
- `drawImage()` solo funciona correctamente en un {{domxref("HTMLVideoElement")}} cuando su {{domxref("HTMLMediaElement.readyState")}} es mayor que 1 (es decir, el evento de búsqueda se activa después de configurar la propiedad `currentTime).
```

File: files/es/web/api/canvasrenderingcontext2d/drawimage/index.md [hunk 6]
```diff
@@ -119,18 +119,15 @@ window.addEventListener("load", drawCanvas);
...
+- `drawImage()` siempre utilizará del elemento de origen _el tamaño intrínseco  en píxeles de CSS_ al dibujar, recortar o escalar.
```
Comment (Graywolf9) [inline files/es/web/api/canvasrenderingcontext2d/drawimage/index.md:129]:
```suggestion
- `drawImage()` siempre utilizará del elemento de origen _el tamaño intrínseco en píxeles de CSS_ al dibujar, recortar o escalar.
```

**Review by Graywolf9 — APPROVED** [review]
Thank you so much!

**Pull request merged**